docs/course_materials/exercises/04_Compound_data_types/class_exercise_1.md

```diff
@@ -0,0 +1,10 @@
+### Let's Practice Together \#1 
+
```

Comment:
Remove this extra line and the presentation will work correctly (with the background on the first slide)